mlflow/server/js/src/i18n/default/en.json

```diff
@@ -611,6 +611,10 @@
     "defaultMessage": "Registered Models",
     "description": "Text for registered model link in the title for model comparison page"
   },
+  "UEDu0c": {
+    "defaultMessage": "MLflow UI automatically fetches metric histories for active runs and updates the metrics plot with a {interval} second interval.",
```

Comment:
Included `interval` so a user doesn't need to guess or measure how long the interval is.